src/store/create-invitation/saga.test.ts

```diff
@@ -39,12 +38,12 @@ describe('fetchInvite', () => {
         storeState: { createInvitation },
       } = await expectSaga(fetchInvite)
         .withReducer(rootReducer, {
-          createInvitation: { code: 'something', url: 'url', invitesUsed: 2, maxUses: 3, isLoading: false },
+          createInvitation: { code: 'something', url: 'url', invitesUsed: 2, inviteCount: '3', isLoading: false },
```

Comment:
is inviteCount from the server actually a string? Should be an integer, I think.

Reply:
Double checked and its number. Updated.